### ISSUE TITLE
Unable to downgrade or change versions after upgrading to Parity 1.9.[0,1,2]-beta

### ISSUE THREAD

**Issue by ltfschoen** [paritytech/parity#7795]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: 1.8.7-stable (the latest version that I can get to work)
> - **Which operating system?**: MacOS
> - **How installed?**: via installer
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: foundation
> - **Did you try to restart the node?**: yes, I closed Parity before each upgrade/downgrade, and then re-opened Parity

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

I was using 1.8.7-stable that I had downloaded via the installer without any issues. I then downloaded 1.9.0-beta "Velocity", but the wallet did not load, so I closed Parity and downgraded by re-downloading 1.8.7-stable via the installer and running it, and the downgrade worked, but whilst it reverted back to the old UI, it kept the new white/grey logo from 1.9.0-beta "Velocity", which I thought was strange behaviour.

Today I upgraded to 1.9.2-beta, but the wallet did not load in that version either, so I closed Parity and downgraded to 1.8.9-stable instead, when it opened Parity after the installation completed, it showed the new UI of 1.9.x, and in the UI it said I was still using 1.9.2-beta, and showed the same in the CLI when I ran `parity --version`, and it has kept new white/grey logo from the 1.9.x version.

So I tried downgrading to 1.8.8-stable instead, but that showed I was still using 1.9.2-beta too.

Only after I downgraded to 1.8.7-stable did it the downgrade actually work such that when I open Parity it shows the old UI of 1.8.x and shows in the footer and in the CLI that i'm using 1.8.7-stable.

But then I decided to try upgrading from 1.8.7-stable to 1.8.8-stable or 1.8.9-stable. I tried to upgrade to each of them, but it doesn't work, it still says I'm using 1.8.7-stable.

**Comment by gaia** [comment]
did you stop parity and try
`rm -Rf /usr/bin/parity`
between reinstalls?

**Comment by ltfschoen** [comment]
I don't have a Parity executable in /usr/bin/parity or /usr/local/bin/parity
When I run `which parity` to find out where the Parity Application is installed it returns /Applications/Parity Ethereum.app/Contents/MacOS/parity.
All my keys and the blockchain are stored in /Users/Username/Library/Application Support/io.parity.ethereum
I just had to drag the Parity Ethereum application to the Trash, and then tried installing Parity 1.8.9-stable, and it worked! Thanks for your guidance!

**Comment by mixedrlt** [comment]
I have somewhat the same problem but from a different version.
I had installed 1.10.2 version with auto_update config set to "critical". Yesterday parity auto updated to 1.11.3. Because of other bugs the parity process just froze so i wanted to downgrade to 1.10.4. When i call parity --version i get 1.10.4, when parity starts i see Starting Parity/v1.11.3. Tried repeatedly to downgrade but nothing and yes i tried also
rm -Rf /usr/bin/parity
With no luck. Why isn't it downgrading? Or what do i have to do?

**Comment by Tbaut** [comment]
What does `which parity` gives?

**Comment by mixedrlt** [comment]
When i start parity i see in logs Starting Parity/v1.11.3
When i say parity version i get 1.10.4
When i say which parity i get /usr/bin/parity
When i downgrade with dpkg i see from (1.10.4) to (1.10.2)
So only when i start parity i see 1.11.3 and it does not matter what i do (downgrade/upgrade or rm -Rf on parity folder)

**Comment by Tbaut** [comment]
what about `parity --force-direct` ?

**Comment by mixedrlt** [comment]
Parity force direct Parity/v1.10.6-stable

**Comment by Tbaut** [comment]
The upgraded version is in a separate folder. To force the older version not to use it, please try to move the updates directory with:
`mv ~/.local/share/io.parity.ethereum-updates/ ~/.local/share/io.parity.ethereum-updates-backup/` 

Then upgrade parity with the [one-line installer](https://wiki.parity.io/Setup#one-line-binary-installer): `bash <(curl https://get.parity.io -L) stable` to upgrade to 1.10.6 (with the bug fix)

If things work as expected now and you can remove safely the backup directory:
`rm -r  ~/.local/share/io.parity.ethereum-updates-backup/`
